### DISCUSSION TITLE
Is Laravel the happiest developer community on the planet?

### DISCUSSION THREAD

**Discussion by mikethemelanson** [community/community#63475] [Discover: GitHub Best Practices]
# ![Screen Shot 2020-12-03 at 10 24 07 AM](https://user-images.githubusercontent.com/8250055/101071884-0efd3e00-3552-11eb-8523-11305d3cc848.png)

:wave: Hey y'all! 

@mikethemelanson with GitHub's [The ReadME Project](https://github.com/readme?src=rmpcod) here again to let you know about our August release. 

This month, @klintron dives into [Laravel](https://github.com/laravel), the highly-opinionated PHP framework with first-party support for a wide range of features, and [asks if it's the happiest developer community on the planet](https://github.com/readme/featured/laravel-community?src=rmpcod). Though many popular projects lose their luster over time, Laravel is still a fan favorite, with the community in a seemingly perpetual honeymoon period. How does the Laravel community maintain such a beloved platform? By ignoring trends and sweating the details. 

We also have a guide from @Drupal Core Accessibility Maintainer Mike Gifford (@mgifford), who argues that [it's essential to treat accessibility issues as bugs, not mere feature requests](https://github.com/readme/guides/fix-accessibility-bugs). And while we're talking about accessibility, this month's developer story is a profile of Anton Mihorodchenko (@GreenWizard2015), a Ukrainian developer with cerebral palsy who [uses AI to not only better communicate and code](https://github.com/readme/stories/anton-mirhorodchenko), but is also using AI to build adaptive tooling for other developers and people with disabilities. Anton's story also includes the latest in GitHub's _[Coding Accessibility](https://www.youtube.com/playlist?list=PL0lo9MOBetEGvxp03Kh3eIlFvIvfE21KF)_ video series, so head on over and give it a watch!

### Taking time to build what's next

Over the next few months The ReadME Project team is focusing on new content, new formats, and new ways to serve you as an audience. 

As we take the time to dream and build, we want your feedback! Tell us what you would love to see! Who would you love to learn from? What content would make your engineering life better? Your feedback is incredibly valuable.

Drop us a line at [thereadmeproject@github.com](mailto:thereadmeproject@github.com) to let us know how YOU want to see The ReadME Project evolve. As always, follow us @github on [LinkedIn](https://www.linkedin.com/company/github/), [Instagram](https://www.instagram.com/github/), [Facebook](https://www.facebook.com/GitHub/), and [Twitter](https://www.twitter.com/github) for updates, polls, and even more opportunities to connect and engage with the open source community on GitHub. And check your inbox for special newsletters, spotlights, and new content over the coming months.

**Comment by Turuk3100** [comment]
Good

**Reply by kimmi-poo** [reply]
Amazing ideas

**Comment by rhino701** [comment]
good. 👍

**Comment by 5DMax** [comment]
Laravel, the popular PHP framework known for its elegant syntax and developer-friendly features, has garnered a reputation not only for its technical prowess but also for its vibrant and enthusiastic community. While it might be a stretch to definitively proclaim it as "the happiest developer community on the planet," there are certainly several aspects that contribute to the positivity and satisfaction within the Laravel community.

**Reply by mikethemelanson** [reply]
Since you're a Laravel developer yourself, @64Dev , what are some of these aspects that you see contributing to the positivity? And is there a different community you'd nominate as the happiest? :octocat:

**Reply by yeikel** [reply]
I think that claiming any community as the "happiest" is not appropriate and dismissive of other communities . How did you measure that and how many communities did you survey?  Saying that it is a happy community is enough. There is no need in my opinion, to claim it as the "happiest" and more so if there is no data to back that up

**Comment by jeevanism** [comment]
Indeed, Laravel makes life easier !

**Reply by mikethemelanson** [reply]
It's been a while since I've written anything in PHP, but hearing about Laravel certainly makes me want to try it out again! I was building small, custom content management systems using PHP back in 2002, and I'd be curious to see how much has changed (a lot, I know!) and what it's like now!

What's your favorite part, @jeevanism ?

**Comment by nirav-gajera** [comment]
Laravel's syntax is very clean and easy to read. This makes it a pleasure to write code in Laravel.
 Laravel comes with a lot of powerful features, such as routing, authentication, and templating. This makes it easy to build complex applications with Laravel.
 it has extensive documentation that covers everything from installation to advanced features. This makes it easy to learn Laravel and get started building applications.

**Comment by billishe** [comment]
Whenever I start a new project with laravel I feel like 40% of the job is already done, the next 10% when I customize and configure it the rest 50% is the actual application. Why would I waste my time setting up repetitive tasks and worrying about non-functional aspects of the app before I actually see an MVP of it. With laravel you can be sure the boilerplate setup is clean, following standards, tested well and reliable allowing you to quickly bootstrap a powerful starting point before adding feature number one! You can't just stop loving it.

**Comment by ghost** [comment]
Laravel's community is known for being active and supportive.

**Comment by mrtripping** [comment]
I remember that when I learned PHP, it was once learning Laravel and I liked it a lot, I used it for several recruitment tests and they worked for me.

**Comment by ssayres** [comment]
im working with laravel for about a year now, and i can tell it is developer-friendly indeed

**Comment by iskanderisker** [comment]
Thanks communtiy guide lines please login this Account page

**Comment by iskanderisker** [comment]
Dear tiktok team I believe my Account was supposed in error because I never violated any communtiy guide lines or I never uproad adult scam nude harmful video in my Account some users must have flagged my videos/Account rendomly and my tiktok impropend/reinstet Account and oblige me thereby

**Comment by iskanderisker** [comment]
![Screenshot_20230802-172011_TikTok](https://github.com/community/community/assets/125094624/d1fa1088-1134-4468-95b8-c98f7e6c7806)
![Screenshot_20230809-042153_TikTok Lite](https://github.com/community/community/assets/125094624/d1817c50-e0eb-4d36-a1f5-151b53eaca46)
![Screenshot_20230803-195821_TikTok Lite](https://github.com/community/community/assets/125094624/ae62c7e1-02d8-4b03-9df2-77dbeeff1d1a)

**Comment by iskanderisker** [comment]
please help me this Account login App

**Comment by laurentiucozma12** [comment]
After I learned 1-year php, I decided this summer to start my first Laravel project and I can say it's a lot better going for a framework. I already deployed the project, so I can say I am pretty happy using Laravel :)

**Comment by armintork** [comment]
> # ![Screen Shot 2020-12-03 at 10 24 07 AM](https://user-images.githubusercontent.com/8250055/101071884-0efd3e00-3552-11eb-8523-11305d3cc848.png)
> 👋 Hey y'all!
> 
> @mikethemelanson with GitHub's [The ReadME Project](https://github.com/readme?src=rmpcod) here again to let you know about our August release.
> 
> This month, @klintron dives into [Laravel](https://github.com/laravel), the highly-opinionated PHP framework with first-party support for a wide range of features, and [asks if it's the happiest developer community on the planet](https://github.com/readme/featured/laravel-community?src=rmpcod). Though many popular projects lose their luster over time, Laravel is still a fan favorite, with the community in a seemingly perpetual honeymoon period. How does the Laravel community maintain such a beloved platform? By ignoring trends and sweating the details.
> 
> We also have a guide from @drupal Core Accessibility Maintainer Mike Gifford (@mgifford), who argues that [it's essential to treat accessibility issues as bugs, not mere feature requests](https://github.com/readme/guides/fix-accessibility-bugs). And while we're talking about accessibility, this month's developer story is a profile of Anton Mihorodchenko (@GreenWizard2015), a Ukrainian developer with cerebral palsy who [uses AI to not only better communicate and code](https://github.com/readme/stories/anton-mirhorodchenko), but is also using AI to build adaptive tooling for other developers and people with disabilities. Anton's story also includes the latest in GitHub's _[Coding Accessibility](https://www.youtube.com/playlist?list=PL0lo9MOBetEGvxp03Kh3eIlFvIvfE21KF)_ video series, so head on over and give it a watch!
> 
> ### Taking time to build what's next
> Over the next few months The ReadME Project team is focusing on new content, new formats, and new ways to serve you as an audience.
> 
> As we take the time to dream and build, we want your feedback! Tell us what you would love to see! Who would you love to learn from? What content would make your engineering life better? Your feedback is incredibly valuable.
> 
> Drop us a line at [thereadmeproject@github.com](mailto:thereadmeproject@github.com) to let us know how YOU want to see The ReadME Project evolve. As always, follow us @github on [LinkedIn](https://www.linkedin.com/company/github/), [Instagram](https://www.instagram.com/github/), [Facebook](https://www.facebook.com/GitHub/), and [Twitter](https://www.twitter.com/github) for updates, polls, and even more opportunities to connect and engage with the open source community on GitHub. And check your inbox for special newsletters, spotlights, and new connt over the coming months.

**Comment by zmemon** [comment]
The sentiment that Laravel has the "happiest developer community on the planet" is a perception that has been echoed by various members and observers of the Laravel community. Here are some insights into this view:

**Positive Community Sentiment:** There is a narrative within the Laravel community that suggests it is indeed very positive and happy. This sentiment is shared within the community itself and is a part of its identity​

**Supportive Ecosystem:** Laravel's ecosystem is described as growing and supportive, with numerous developers sharing their knowledge and experiences. This culture of sharing and support can significantly contribute to overall community happiness​.

**Community and Support:** The popularity and usage of Laravel have increased over the years, creating a large and active community. This community is characterized by its readiness to provide support, and developers often receive quick responses to issues, which could contribute to a sense of satisfaction and happiness​.

**Sustained Enthusiasm:** Discussion on platforms like GitHub suggests that the Laravel community maintains a "perpetual honeymoon period," indicating a sustained enthusiasm for the framework. Such enthusiasm can be infectious and contribute to a positive community atmosphere

**Comment by adnanenet** [comment]
Honestly, the best framework now is Django, not Laravel. What I hate about Laravel is that I just started learning version 8, and then version 9 is coming. It's not really stable, and I see PHP as a language I find frustrating 👍 .

**Comment by supuna97** [comment]
Laravel is a powerful PHP framework that allows you to build web applications quickly and efficiently. One of the ways to make your Laravel application more object-oriented is to use object-oriented programming (OOP) techniques to organize your code. 
It is very easy to manipulate the REST APIs, database migrations and middleware as you need.

**Comment by JekugaBrandon** [comment]
are companies still using PHP for app?

**Reply by mikethemelanson** [reply]
Wordpress is just one example. :)

**Comment by ImanAdithya** [comment]
The statement that Laravel has the "happiest developer community on the planet" is subjective and may vary from person to person. Laravel is a PHP web application framework known for its elegant syntax, developer-friendly features, and a vibrant ecosystem. The framework has gained popularity for its ease of use and the way it simplifies common tasks in web development.

However, whether a developer community is the happiest depends on individual preferences, experiences, and needs. Different developers may find happiness and satisfaction in various communities based on factors such as the framework's documentation, community support, ease of learning, and the specific needs of their projects.

Laravel does have a strong and supportive community, providing resources, tutorials, and active forums. The Laravel creator, Taylor Otwell, has played a significant role in fostering a positive and inclusive environment.

Ultimately, the happiness of developers within a community is subjective and can vary based on personal preferences, project requirements, and the development ecosystem that best aligns with their goals.

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules
